hack/release-helper.sh

```diff
@@ -34,3 +36,9 @@ find deployment/helm -name Chart.yaml | xargs -I '{}' \
 find deployment/helm -name values.yaml | xargs -I '{}' \
     sed -e s"/pullPolicy:.*/pullPolicy: IfNotPresent/" -i '{}'
 
+# Patch deployment templates
+echo Patching kustomize templates to use $container_image
+find deployment/operator/config/manifests/bases -name nri-plugins-operator.clusterserviceversion.yaml | xargs -I '{}' \
+    sed -E -e s",^([[:space:]]+)containerImage:.+$,\1containerImage: $container_image," -i '{}'
+find deployment/operator/config/manager -name kustomization.yaml | xargs -I '{}' \
+    sed -E -e s",^([[:space:]]+)newTag:.+$,\1newTag: $version," -i '{}'
```

Comment:
So these inline-[s]edits would be IMO better to replace with [s]editing `.yaml.in`s into corresponding `.yaml`s to keep these actions idempotent wrt. to both the repo content and state.

Reply:
@fmuyassarov And with #224 in place, you'd need to do something like this instead:

```
...
make -C deployment/operator kustomizations && \
    find deployment/operator/config/manager -name kustomization.yaml | xargs -I '{}' \
        sed -E -e s",^([[:space:]]+)newTag:.+$,\1newTag: $version," -i '{}'
```